### ISSUE TITLE
Failed to restart cron.service: Unit not found. | MariaDB not restart with all the services.

### ISSUE THREAD

**Issue by locksmithunit** [engintron/engintron#1381]
Hey @fevangelou 
I know it's 3 tickets in a row,
but it's better to get over with that one time,
**I organize the topics, It will be easier for everybody.**

So, i notice from V2 the cron service failed to restart when
i **"Restart all the essential services"** button throw Engintron.
```
=========================================
=== Restarting All Important Services ===
=========================================

Restarting Cron...
Redirecting to /bin/systemctl restart cron.service
Failed to restart cron.service: Unit not found.

Restarting CSF...
Flushing chain `INPUT'
Flushing chain `FORWARD'
Flushing chain `OUTPUT'
Flushing chain `ALLOWIN'
Flushing chain `ALLOWOUT'
Flushing chain `DENYIN'
Flushing chain `DENYOUT'
Flushing chain `INVALID'
Flushing chain `INVDROP'
Flushing chain `LOCALINPUT'
Flushing chain `LOCALOUTPUT'
Flushing chain `LOGDROPIN'...

```

If you look at the start of the "restart log" i sent here, (Throw the Engintron interface), you will see the cron.service failed to restart.
It may be MASKED or something, but is not working.

Another thing,
I said it last time,
It is MariaDB not restarting with all the services.
( not appearing on the "restart log" as well. )

What to do? waiting for Engintron 2.1?
Something to fix it?
Again, My Engintron it's on Default and the cPanel is default as well.

Waiting for your help,
Happy Christmas.

**Comment by fevangelou** [comment]
Update once more, it's been fixed in build 20220103. The service is "crond" and not "cron". It was a typo.

**Comment by locksmithunit** [comment]
Magic :)
Thank you @fevangelou
